src/common/components/testimonial/index.js

```diff
@@ -6,7 +6,9 @@ export default function Testimonial(props) {
   return (
     <div>
       <blockquote className={ style.pullQuote }>
-        <p>{ props.children }</p>
+        <p>
+          <span>&ldquo;</span>{ props.children }<span>&rdquo;</span>
```

Comment:
As mentioned in CSS comment - there shouldn't be a need for quotes in HTML as they can be added styled in CSS.